pkg/webhook/default_server/sidecarset/mutating/sidecarset_create_update_handler.go

```diff
@@ -57,10 +62,25 @@ type SidecarSetCreateHandler struct {
 	Decoder types.Decoder
 }
 
-func setDefaultSidecarSet(sidecarset *appsv1alpha1.SidecarSet) {
+func  setDefaultSidecarSet(sidecarset *appsv1alpha1.SidecarSet) {
+	setSidecarSetStratety(&sidecarset.Spec.Strategy)
+
 	for i := range sidecarset.Spec.Containers {
 		setDefaultContainer(&sidecarset.Spec.Containers[i])
 	}
+
+	klog.V(3).Infof("sidecarset after mutating: %v", util.DumpJSON(sidecarset))
+}
+
+func setSidecarSetStratety(strategy *appsv1alpha1.SidecarSetStrategy) {
```

Comment:
Rename to setDefaultSidecarSetUpdateStrategy?

Reply:
done